subsys/logging/Kconfig

```diff
@@ -23,6 +23,8 @@ rsource "Kconfig.filtering"
 
 rsource "Kconfig.processing"
 
+rsource "Kconfig.nrfx"
```

Comment:
Platform specific files should not be located in `subsys/logging`. It would be better suited in `modules/hal_nordic`